scripts/go-mod-tidy-all.sh

```diff
@@ -19,7 +19,7 @@ then
     if ! command -v gomod2nix &> /dev/null
     then
         echo "gomod2nix could not be found in PATH, installing..."
-        go install github.com/nix-community/gomod2nix@latest
+        go install github.com/nix-community/gomod2nix@872b63ddd28f318489c929d25f1f0a3c6039c971
```

Comment:
this script need to change a little bit:
```bash
if command -v nix &> /dev/null
then
    nix develop .. -c gomod2nix
else
    if ! command -v gomod2nix &> /dev/null
    then
        # install gomod2nix manually
        echo "gomod2nix could not be found in PATH, installing..."
        go install github.com/nix-community/gomod2nix@master
    fi
    gomod2nix
fi
```

Reply:
the `nix develop` version should be more stable, because it also setup correct go version according to configs in `go.mod`.

Reply:
Awesome, thank you!  I'll get your changes in in a little bit :)